### PR TITLE
005 expected competencies per band

### PR DESCRIPTION
**Pull request by maciekkaczor**

### CODE REVIEW

File: app/competencyData.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const axios = require('axios')
+
+exports.getCompetencies = async (bandName) => { 
+      try {  
+        const competenciesResponse = await axios.get('http://localhost:8080/api/band-competencies/' + bandName)
+        return competenciesResponse.data
+      } catch (e) {
+         return new Error('Could not get a response from the API with the band competencies')
+
```
Comment (maxlilleyman) [inline app/competencyData.js]: Remove empty line

File: app/routes.js [hunk 2]
```diff
@@ -16,3 +17,14 @@ router.get('/jobRoles', async (req, res) => {
   });
 
 module.exports = router
+
+router.get('/competencies/:bandName', async (req, res) => {
+  var bandName = req.params.bandName;
+  res.render('competenciesView', {
+    competencies: await competencyData.getCompetencies(bandName)
```
Comment (maxlilleyman) [inline app/routes.js]: Remove bandName declaration and have getCompetencies reference req.params.bandName

File: app/views/competenciesView.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      {% for i in range (1, headerArray.length) %}
+        <tr>
+          <td class="govuk-table__cell">
+            <p><b>{{ headerArray[i] }}</b></p>
+            <p>{{ bodyArray[i] }}</p>
+          </td>
+        </tr>
+      {% endfor %}
+    {% endif %}
+
```
Comment (maxlilleyman) [inline app/views/competenciesView.html]: Remove empty line

File: app/routes.js (hunk 2 above)
Comment (maciekkaczor) [reply app/routes.js]: fixed

File: app/competencyData.js (hunk 1 above)
Comment (maciekkaczor) [reply app/competencyData.js]: fixed

File: app/views/competenciesView.html (hunk 3 above)
Comment (maciekkaczor) [reply app/views/competenciesView.html]: fixed

**Review by maxlilleyman — APPROVED** [review]

File: app/views/competenciesView.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% extends "layout_unbranded.html" %}
+
+{% block pageTitle %}
+  {{ serviceName }}
+{% endblock %}
+
+{% block content %}
+{% if competencies and competencies.length > 0 %} 
+<h2>{{competencies[0].name}} competencies</h2>
```
Comment (michaelkerrigan) [inline app/views/competenciesView.html]: Could you look at the way the title is done on the homeView.html page on the Main branch? It was changed to make it look a little cleaner

File: app/views/competenciesView.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+          </td>
+        </tr>
+      {% endfor %}
+    {% endif %}
+  {% endfor %}
+  </tbody>
+</table>
+
+{% else %}
+There are no competencies.
```
Comment (michaelkerrigan) [inline app/views/competenciesView.html]: Same again, there have been a few changes to this line in the Main branch. For consistency could you check you're doing it the same way?

**Review by michaelkerrigan — CHANGES_REQUESTED** [review]

File: app/views/competenciesView.html (hunk 4 above)
Comment (maciekkaczor) [reply app/views/competenciesView.html]: fixed

File: app/views/competenciesView.html (hunk 5 above)
Comment (maciekkaczor) [reply app/views/competenciesView.html]: fixed

**Review by michaelkerrigan — APPROVED** [review]

**Review by adamlelliott — APPROVED** [review]

**Review by boydstepnew — APPROVED** [review]

**Pull request merged**